### ISSUE TITLE
Binary Classifier experiment - log loss

### ISSUE THREAD

**Issue by andrewcz** [fchollet/keras#2007]
Hi,
Fantastic Library!
I was just wondering, i am trying to use the library for a binary classifier experiment using the log loss function to train the model. This is for a university experiment around benchmarking different models. Would you have time to provide an example of how to use the library to achieve the above goal.
Also, with an included visualisation on how the model learns.
Many thanks,
Best,
Andrew

**Comment by jerheff** [comment]
An example of binary classification is here:  http://keras.io/examples/
"MLP for binary classification"
